runtime/android/core_internal/src/org/xwalk/core/internal/XWalkPresentationHost.java

```diff
@@ -28,7 +28,7 @@
     private static Activity sEnduringActivity = null;
     private static XWalkPresentationHost sInstance;
 
-    public class RenderFrameHostId {
+    public static class RenderFrameHostId {
```

Comment:
Because it's now used (new XXX) in a static method and there is no implicit this object anymore :) And static class in Java can access less member of the outer class, making it clearer...